corehq/form_processor/serializers.py

```diff
@@ -53,6 +54,25 @@ class Meta:
         exclude = ('id',)
 
 
+class HumanReadableChoiceField(serializers.ChoiceField):
+    def get_attribute(self, obj):
+        choice = super(serializers.ChoiceField, self).get_attribute(obj)
+        return self.choices[choice]
+
+
+class DeletableModelWithJsonSerializer(DeletableModelSerializer):
+    serializer_field_mapping = {}
+    serializer_field_mapping.update(DeletableModelSerializer.serializer_field_mapping)
+    serializer_field_mapping[JSONField] = serializers.DictField
```

Comment:
Oh and I forgot to mention that updating this mapping will automatically convert all `JSONField`s appropriately without having to manually list each one.